packages/lib/convertToNewDurationType.ts

```diff
@@ -0,0 +1,25 @@
+export default function convertToNewDurationType(prevType: string, newType: string, prevValue: number) {
```

Comment:
NIT:
Declare any integer as an exportable const with a definition as its name and reuse it. Probably we already had them.

export const MINUTES_IN_HOUR = 60;
export const MINUTES_IN_DAY = 1440;
export const HOURS_IN_DAY = 24;